Source/WebCore/dom/Attr.cpp

```diff
@@ -101,9 +102,31 @@ ExceptionOr<void> Attr::setPrefix(const AtomString& prefix)
 
 ExceptionOr<void> Attr::setValue(const AtomString& value)
 {
-    if (RefPtr element = m_element.get())
-        return element->setAttribute(qualifiedName(), value, true);
-    else
+    if (RefPtr element = m_element.get()) {
+        auto verifiedValue = value;
+        if (document().requiresTrustedTypes()) {
+            AttributeTypeAndSink attributeTypeAndSink = trustedTypeForAttribute(element->nodeName(),
+            qualifiedName().localName(), element->namespaceURI(), qualifiedName().namespaceURI());
```

Comment:
Formatting mistake here, but also, doesn’t need such a long name. The type system makes things clear enough for a local variable.
```suggestion
            auto type = trustedTypeForAttribute(element->nodeName(),
                qualifiedName().localName(), element->namespaceURI(), qualifiedName().namespaceURI());
```